### PR TITLE
Add citeproc

### PR DESCRIPTION
**Pull request by mohit038**

### CODE REVIEW

**Comment by tobiasdiez** [comment]
What's the status here? It seems like the current main branch already includes some of the changes.

File: src/Utils/ProtectedRoutes.tsx [hunk 1]
```diff
@@ -17,7 +17,11 @@ function ProtectedRoutes({ children, ...rest }: ProtectedRoutesProps) {
     <Route
       {...rest}
       render={({ location }) =>
+<<<<<<< HEAD
```
Comment (Siedlerchr) [inline src/Utils/ProtectedRoutes.tsx:20]: Merge conflict?

**Comment by Siedlerchr** [comment]
You should rebase on the latest main,/merge in the latest main yesterday I updated the

**Comment by mohit038** [comment]
Added these changes in #103

**Pull request closed without merging**